ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -200,4 +213,9 @@ message PartyDetails {
   // Optional,
   // Modifiable
   ObjectMeta local_metadata = 4;
+
+  // The id of the ``Identity Provider``
+  // If not provided, it would be assumed a default identity provider config or party is not hosted by the participant.
+  // Optional
```

Comment:
Is this modifiable or not?

Reply:
I thought not, but after conversation with @mziolekda we agreed to make it modifiable by participant admin